src/components/data.js

```diff
@@ -0,0 +1,31 @@
+import {BsPeopleFill} from "react-icons/bs"
```

Comment:
All these imports are imported from the same library so could have been one import